### PR TITLE
ci: use bullseye image in CI

### PR DESCRIPTION
**Pull request by isaranto**
Using bullseye image for CI. In this PR we install the additional dependencies required to run tests etc using Debian Bullseye which is used by inference-services.
Changes:

- CI commands exist in Makefile so that we can easily transfer them to any platform
- updated dictionaries that have a hunspell package in debian. hunspell is a "superset" of myspell. Will also change these in inference-services image.
- fixed failing tests due to changes in language dictionaries
- added np.random.seed for test that failed randomly due to the values hardcoded in the test
- skipped test that is failing in CI for serbian language - to revisit with the python upgrade

### CODE REVIEW

**Review by kevinbazira — COMMENTED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: "CI"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  ci-merge:
+    runs-on: ubuntu-latest
```
Comment (elukey) [inline .github/workflows/ci.yml:8]: Nit: Is it a leftover? Should it be bullseye?

**Comment by elukey** [comment]
Left a nit but the rest LGTM, merge when you prefer. Thanks for this work!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (isaranto) [reply .github/workflows/ci.yml:8]: No it is not a leftover. The available options for github runners are macos, winows and ubuntu ([available runners](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on)).
So we use an ubuntu machine but run our things inside a container that runs the bullseye image.

**Review by elukey — DISMISSED** [review]

**Review by elukey — APPROVED** [review]

**Pull request merged**